webapp/public/locales/en/main.json

```diff
@@ -117,7 +118,14 @@
   "form.field.minValue": "The minimum value is {{0}}",
   "form.field.maxValue": "The maximum value is {{0}}",
   "form.field.notAllowedValue": "Not allowed value",
-  "form.field.specialChars": "Special characters allowed: {{0}}",
+  "form.field.allowedChars": "Special characters allowed: {{0}}",
+  "form.field.specialCharsNotAllowed": "Special characters are not allowed",
+  "form.field.spacesNotAllowed": "Spaces are not allowed",
+  "form.field.requireLowercase": "Password must contain at least one lowercase letter.",
+  "form.field.requireUppercase": "Password must contain at least one uppercase letter.",
+  "form.field.requireDigit": "Password must contain at least one digit.",
+  "form.field.requireSpecialChars": "Password must contain at least one special character.",
```

Comment:
these keys are specific to password field 
TODO: rename the key to be more specific like `form.field.password.requireLowercase`